activerecord/lib/active_record/encryption/encrypted_attribute_type.rb

```diff
@@ -135,6 +135,13 @@ def decryption_options
         def clean_text_scheme
           @clean_text_scheme ||= ActiveRecord::Encryption::Scheme.new(downcase: downcase?, encryptor: ActiveRecord::Encryption::NullEncryptor.new)
         end
+
+        def deserialize_previous_value_to_determine_change(raw_old_value)
+          deserialize(raw_old_value)
+          # We tolerate unencrypted data when determining if a column changed to support default DB values in encrypted attributes
```

Comment:
Can you wrap this to 80 characters.

```suggestion
          # We tolerate unencrypted data when determining if a column
          # changed to support default DB values in encrypted attributes.
```